### PR TITLE
[Smart Chain] Add config

### LINKED ISSUE #1191: [Binance Smart Chain] Add transaction api
It's Ethereum based

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1191

### CODE REVIEW

File: coin/coins.go [hunk 1]
```diff
@@ -578,6 +580,16 @@ var Coins = map[uint]Coin{
 		MinConfirmations: 0,
 		SampleAddr:       "erd12tqtt5zcg6vpw65y4hkanvt49kzq695sr3ctuszjy92xw0ppzcssy2xd5r",
 	},
+	BSC: {
+		ID:               10000714,
+		Handle:           "bsc",
+		Symbol:           "BNB",
+		Name:             "BSC",
```
Comment (EnoRage) [inline coin/coins.go:587]: Binance smart chain?

File: coin/coins.go [hunk 2]
```diff
@@ -578,6 +580,16 @@ var Coins = map[uint]Coin{
 		MinConfirmations: 0,
 		SampleAddr:       "erd12tqtt5zcg6vpw65y4hkanvt49kzq695sr3ctuszjy92xw0ppzcssy2xd5r",
 	},
+	BSC: {
+		ID:               10000714,
+		Handle:           "bsc",
```
Comment (EnoRage) [inline coin/coins.go:585]: binancesmartchain?

File: coin/coins.go [hunk 3]
```diff
@@ -578,6 +580,16 @@ var Coins = map[uint]Coin{
...
+		Symbol:           "BNB",
```
Comment (EnoRage) [inline coin/coins.go:586]: Cannot see preferredSymbol

Comment (hewigovens) [reply coin/coins.go:586]: added

File: coin/coins.go (hunk 2 above)
Comment (hewigovens) [reply coin/coins.go:585]: this one is defined as `bsc` in wallet core, suggests by Viktor

**Review by EnoRage — APPROVED** [review]

**Pull request merged**